docs/monorepo.md

```diff
@@ -0,0 +1,79 @@
+
+Our mirroring tooling also uses `.gitattributes` to specify built files to include in the mirror and unnecessary files to exclude.
+
+### Testing
+
+TBD. Currently we only have semi-generic testing set up for packages via `.scripts.phpunit` in `composer.json`, with an override via `tests/php/ci-can-run.sh` existing and exiting with a failure status.
+
+### Mirror repositories
+
+Most projects in the monorepo should have a mirror repository holding a built version of the project, ready for deployment. Follow these steps to create the mirror repo and configure the monorepo tooling to push to it.
+
+1. Create the mirror repo on GitHub. It will most likely be named like "https://github.com/Automattic/jetpack-_something_".
+   1. The repo's description should begin with `[READ ONLY]` and end with `This repository is a mirror, for issue tracking and development head to: https://github.com/automattic/jetpack`.
+   2. The default branch should be `master`, matching the monorepo.
+   3. In the repo's settings, turn off wikis, issues, projects, and so on.
+   4. Make sure that [matticbot](https://github.com/matticbot) can push to the repo.
+   5. Make sure that Actions are enabled. The build process copies workflows from `.github/files/mirror-.github` into the mirror to do useful things like automatically close PRs with a reference back to the monorepo.
+2. If your project requires building, configure `.scripts.build-production` in your project's `composer.json` to run the necessary commands.
+3. If there are any files included in the monorepo that should not be included in the mirror, use `.gitattributes` to tag them with "production-exclude".
+4. If there are any built files in `.gitignore` that should be included in the mirror, use `.gitattributes` to tag them with "production-include".
+5. Set `.extra.mirror-repo` in your project's `composer.json` to the name of the repo.
+   * When you push the PR making this change to `composer.json`, pay attention to the Build workflow. Download the "jetpack-build" artifact and make sure it contains your project, and that there are no extra or missing files.
+
+### Plugin release tooling
+
+If you have set `.extra.mirror-repo`, `.extra.release-branch-prefix`, and `.extra.wp-plugin-name` in your plugin's `composer.json`, we have tooling to make releasing to WordPress.org easier.
+
+* `tools/create-release-branch.sh` will help you create the correctly named release branch, and will automatically update version numbers and versions of monorepo packages for you. The GitHub Action will then mirror this branch to your plugin's mirror repo.
+* `tools/deploy-to-svn.sh` will prepare a temporary directory with the content of the mirror repo branch that is ready to be pushed to WordPress.org SVN.
+* `tools/revert-release.sh` will prepare a temporary directory that updates the "Stable version" tag in `readme.txt` set to the previous version, in case an emergency rollback is required.
```

Comment:
```suggestion
* `tools/revert-release.sh` will prepare a temporary directory that updates the "Stable version" tag in `readme.txt` to the previous version, in case an emergency rollback is required.
```